src/resources/breaking-changes.md

```diff
@@ -0,0 +1,818 @@
+---
+title: Breaking changes
```

Comment:
I think we should make this cover deprecations as well, so perhaps "Breaking changes and deprecations"?